extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -857,10 +857,13 @@ export class ProjectsController {
 
 		const newFilePath = path.join(path.dirname(utils.getPlatformSafeFileEntryPath(file?.relativePath!)), `${newFileName}.sql`);
 
-		try {
-			await this.move(node, node.projectFileUri.fsPath, newFilePath);
-		} catch (e) {
-			void vscode.window.showErrorMessage(constants.errorRenamingFile(file?.relativePath!, newFilePath, utils.getErrorMessage(e)));
+		const renameResult = await this.move(node, node.projectFileUri.fsPath, newFilePath);
+
+		if (renameResult?.success) {
+			TelemetryReporter.createActionEvent(TelemetryViews.ProjectTree, TelemetryActions.rename).send();
```

Comment:
(same for other similar calls)

```suggestion
			TelemetryReporter.sendActionEvent(TelemetryViews.ProjectTree, TelemetryActions.rename);
```